packages/kbn-router-to-openapispec/src/oas_converter/zod/lib.test.ts

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { z } from 'zod';
+import { is, convert, convertPathParameters, convertQuery } from './lib';
+
+import { createLargeSchema } from './lib.test.util';
+
+describe('zod', () => {
+  describe('convert', () => {
+    test('base case', () => {
+      expect(convert(createLargeSchema())).toEqual({
+        schema: {
+          additionalProperties: false,
+          properties: {
+            any: {
+              description: 'any type',
+            },
+            booleanDefault: {
+              default: true,
+              description: 'defaults to to true',
+              type: 'boolean',
+            },
+            ipType: {
+              format: 'ipv4',
+              type: 'string',
+            },
+            literalType: {
+              enum: ['literallythis'],
+              type: 'string',
+            },
+            map: {
+              items: {
```

Comment:
This snapshot was different between config-schema and zod, please let me know if the map object is not correct.

Reply:
Little bit tricky to judge... Map is a weird type to be using in OAS/route endpoints anyway though!